examples/text_prediction/run_text_prediction.py

```diff
@@ -49,46 +55,66 @@ def get_parser():
     parser.add_argument('--config_file', type=str,
                         help='The configuration of the TextPrediction module',
                         default=None)
+    parser.add_argument('--use_tabular',
+                        action='store_true',
+                        help='Whether to use the stack ensemble '
+                             'enabled in AutoGluon Tabular for this task.')
     return parser
 
 
+def set_seed(seed):
+    import mxnet as mx
+    import torch as th
+    th.manual_seed(seed)
+    mx.random.seed(seed)
+    np.random.seed(seed)
+    random.seed(seed)
+
+
 def train(args):
+    set_seed(args.seed)
     if args.task is not None:
-        feature_columns, label_columns, stop_metric, eval_metrics = TASKS[args.task]
+        feature_columns, label_column, eval_metric, all_metrics = TASKS[args.task]
     else:
         raise NotImplementedError
     if args.exp_dir is None:
-        args.exp_dir = 'autogluon_{}'.format(args.task)
-    model = task.fit(train_data=args.train_file,
-                     label=label_columns,
-                     feature_columns=feature_columns,
-                     output_directory=args.exp_dir,
-                     stopping_metric=stop_metric,
-                     ngpus_per_trial=1,
-                     eval_metric=eval_metrics)
-    dev_metrics_scores = model.evaluate(args.dev_file, metrics=eval_metrics)
-    with open(os.path.join(args.exp_dir, 'final_model_dev_score.json'), 'w') as of:
-        json.dump(dev_metrics_scores, of)
-    dev_prediction = model.predict(args.dev_file)
-    with open(os.path.join(args.exp_dir, 'dev_predictions.txt'), 'w') as of:
-        for ele in dev_prediction:
-            of.write(str(ele) + '\n')
-    model.save(os.path.join(args.exp_dir, 'saved_model'))
-    model = task.load(os.path.join(args.exp_dir, 'saved_model'))
-    test_prediction = model.predict(args.test_file)
-    with open(os.path.join(args.exp_dir, 'test_predictions.txt'), 'w') as of:
-        for ele in test_prediction:
-            of.write(str(ele) + '\n')
+        args.exp_dir = 'autogluon_text_{}'.format(args.task)
+    train_df = load_pd.load(args.train_file)
+    dev_df = load_pd.load(args.dev_file)
+    test_df = load_pd.load(args.test_file)
+    train_df = train_df[feature_columns + [label_column]]
+    dev_df = dev_df[feature_columns + [label_column]]
+    test_df = test_df[feature_columns]
+    if args.use_tabular:
+
+        predictor = TabularPredictor(label=label_column,
+                                     eval_metric=eval_metric,
+                                     path=args.exp_dir,
+                                     presets='multimodal')
```

Comment:
there is no 'multimodal' preset. It is hyperparameters='multimodal'. If you also want stacking, you'd do: presets='best_quality'